timesketch/api/v1/resources/searchtemplate.py

```diff
@@ -100,21 +100,19 @@ def get(self, searchtemplate_id):
         Returns:
             Search template in JSON (instance of flask.wrappers.Response)
```

Comment:
Can you make that maybe with an example on what is returned? E.g. `{"objects": [{"query_string": "*foobar*"}], "meta": {}}`

Reply:
Yeah, it actually returns a search query string. Changed the wording.